docs/reference/ilm/index.asciidoc

```diff
@@ -10,7 +10,11 @@
 --
 beta[]
 
-The <<index-lifecycle-management-api, {ilm} (ILM) APIs>> enable you to automate how you
+WARNING: {Ilm} is not supported by
```

Comment:
Do we want to add a note along the lines of "While ILM is in beta"?

Reply:
I purposefully left this out since I didn't want to set any expectations of things that aren't finalized yet. I figure this will be removed from the `6.x` branch at the appropriate time that it is not needed for the `current` docs. That being said, this is only being merged into `6.x`, and not `master`, with the belief that it will not be applicable then.

Reply:
If we are not going to mention "beta" here can we at least say "currently not supported" to let the user know that this is not the end state and this is something we intend to solve?